bigbone/src/test/assets/notifications.json

```diff
@@ -64,7 +64,7 @@
   },
   {
     "id": "5908530",
-    "type": "admin.report",
+    "type": "reblog",
```

Comment:
Just asking: Why was this rename necessary?

Reply:
In order to fix the test within NotificationMethodsTest

Reply:
@andregasser We are currently [missing a few types](https://github.com/andregasser/bigbone/pull/310#discussion_r1375989169) and should probably add them soon via a follow-up PR.

Reply:
➡️ #313